core/iwasm/libraries/libc-builtin/libc_builtin_wrapper.c

```diff
@@ -884,7 +885,8 @@ __cxa_allocate_exception_wrapper(wasm_exec_env_t exec_env, uint32 thrown_size)
 
 static void
 __cxa_begin_catch_wrapper(wasm_exec_env_t exec_env, void *exception_object)
-{}
+{
+}
```

Comment:
Should not change it, or CI code guideline will report error.

Reply:
Emm, it was changed by clang-format